src/runtime.js

```diff
@@ -821,7 +821,6 @@ define([
     };
 
     FirstPass.prototype.visitComment = function (ast) {
-        //TODO Use RegEx
```

Comment:
If this was intended to increase efficiency i made some tests and is better of using current indexOf:
https://jsperf.com/regexp-test-vs-string-includes-vs-string-indexof
https://jsperf.com/exist-regexp-test-vs-string-includes-vs-string-indexof

If not i can craft some regex later.